src/screens/sketch/sketch.component.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import {
+    View,
+    Text,
+    TouchableHighlight
+} from 'react-native';
+import SketchView from 'react-native-sketch-view';
+
+import { ImageStore } from 'react-native'
+
+const sketchViewConstants = SketchView.constants;
+
+const tools = {};
+
+tools[sketchViewConstants.toolType.pen.id] = {
+    id: sketchViewConstants.toolType.pen.id,
+    name: sketchViewConstants.toolType.pen.name,
+    nextId: sketchViewConstants.toolType.eraser.id
+};
+tools[sketchViewConstants.toolType.eraser.id] = {
+    id: sketchViewConstants.toolType.eraser.id,
+    name: sketchViewConstants.toolType.eraser.name,
+    nextId: sketchViewConstants.toolType.pen.id
+};
+
+class HandNote extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            toolSelected: sketchViewConstants.toolType.pen.id
+        };
+    }
+
+    isEraserToolSelected() {
+        return this.state.toolSelected === sketchViewConstants.toolType.eraser.id;
+    }
+
+    toolChangeClick() {
+        this.setState({ toolSelected: tools[this.state.toolSelected].nextId });
+    }
+
+    getToolName() {
+        return tools[this.state.toolSelected].name;
+    }
+
+    onSketchSave(saveEvent) {
+        this.props.onSave && this.props.onSave(saveEvent);
```

Comment:
Curto circuito é uma técnica usada por devs mais experientes e ela é um tanto polemica... 
Como nosso time como um todo não é tão avançado assim em JS e o codeclimate não gostou, coloque um if mesmo. 
if (this.props.onSave) this.props.Save(saveEvent)